content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -193,6 +194,11 @@ the following types of volumes:
 * Azure Disk
 * Portworx
 * FlexVolumes
+* CSI - {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+CSI support is in alpha in 1.14 release and requires enabling `ExpandCSIVolumes` feature gate and
```

Comment:
Don't need to repeat that it's alpha since we have the feature tag. But keep the feature gate note.

Reply:
done.